cves/kernel/CVE-2012-2375.yml

```diff
@@ -129,10 +150,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: Did not find any unit tests for this.
+  fix: false
+  fix_answer: Did not find any unit tests for this.
```

Comment:
Reword to include explanation for Linux unit testing (or limitations for unit tests)

Reply:
Updated